app/models/bank_account.rb

```diff
@@ -9,4 +9,6 @@ class BankAccount < ActiveRecord::Base
   validates_format_of :iban, :with => /\A[A-Z]{2}[0-9]{2}[0-9A-Z]{,30}\z/
   validates_numericality_of :balance, :message => I18n.t('bank_account.model.invalid_balance')
 
+  def find_import_method
+  end
```

Comment:
I think this method could use a comment, and raise an exception when there is no valid method (in the manner of AR's find). Or document that it could return `nil`.

Reply:
It appears I was off: [`find_by`](https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by) can return `nil` (but [find](https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find) not).

Still let's document what this method does (including `@return` mentioning `nil` and the return type), also because it's meant to be hooked by plugins (I guess).
And for the moment, `@todo` with CSV would be useful as well.

Reply:
i've added some comments